cmake/modules/platform/os/win.cmake

```diff
@@ -36,3 +36,15 @@ list(APPEND OMR_PLATFORM_DEFINITIONS
 	-D_WIN32_WINNT=0x0601
 	-DWINVER=0x0601
 )
+
+get_filename_component(kit_dir "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows Kits\\Installed Roots;KitsRoot]" REALPATH)
+if(OMR_ENV_DATA64)
+	set(kit_dir "${kit_dir}/bin/x64")
+else()
+	set(kit_dir "${kit_dir}/bin/x86")
+endif()
+# find the message compiler
+find_program(CMAKE_MC_COMPILER mc.exe HINTS "${kit_dir}")
```

Comment:
Could the `kit_dir` be some call that a user could specify if they have it installed in a random location that they do not want to have on their path?

Reply:
no matter where they install it, the location should still be stored in the registry (which is where we pull `kit_dir` from). If for whatever reason its not in the registry you could still specify `-DCMAKE_MC_COMPILER=xxxx` on the command line.